locales/base/translation.json

```diff
@@ -1990,10 +1990,10 @@
     "subtitle": "What would you like to do first?",
     "later": "I'll explore later",
     "options": {
-      "add": "Add funds",
-      "dapp": "Find ways to use crypto",
-      "profile": "Build my profile",
-      "learn": "Learn about Celo"
+      "add": "Add funds to your wallet",
+      "dapp": "Explore earning opportunities",
+      "profile": "Build your profile",
+      "learn": "Learn about Valora Points"
```

Comment:
Gotcha, will keep this in mind for the future changes!
@bakoushin Should I apply this change to any existing hardcoded "Valora" occurences if it's gonna appear in any future tasks?

Reply:
if it's along your way to update, then do it but i think we will have a separate task to make a wider change to make everything use `appName`. this isn't a backwards incompatible change because [appName is a default i18n variable](https://github.com/valora-inc/wallet/blob/main/src/i18n/index.ts#L57)

Reply:
Fixed.